### ISSUE TITLE
How to specify eval_metric of xgboost on yarn under command line version.

### ISSUE THREAD

**Issue by pangguosheng1106** [dmlc/xgboost#3327]
I want to train a regression task by xgboost on yarn under command line version, and run task through the command:
`../../dmlc-core/tracker/dmlc-submit --cluster=yarn --worker-cores=8 --worker-memory=14g --num-workers=20 ../../xgboost mushroom.conf nthread=4 model_dir=hdfs:///user/gspang/data/model`
I've been browsing this document. [https://github.com/dmlc/xgboost/blob/master/demo/binary_classification/README.md](url). 
It just show me how to specify eval_metric from a list in my config file(mushroom.conf). However. I want to make my loss function, and what i need to do to achieve my goal. Thank you. Forget my poor English.

**Comment by tqchen** [comment]
questions are now encouraged at https://discuss.xgboost.ai/ you can specify loss in config file as well
